### ISSUE TITLE
RVL-111 User change without cache clearing

### ISSUE THREAD

**Issue by Rkareko** [OpenSRP/opensrp-client-reveal#738]
Issue documented on [Jira](https://smartregister.atlassian.net/browse/RVL-111)

**Comment by Rkareko** [comment]
Resolved by https://github.com/OpenSRP/opensrp-client-reveal/pull/719

**Comment by Naima-Bashir** [comment]
I just tested this story. I logged in to the opensrp-reveal-release-4.0.0-zm.apk
I used NL1 then CD2 with password Datacollector1

I got the notification asking if I want to clear my data. I clicked okay.

I then got a Login failed error.

I went to check the url and noticed that after it cleared my data; it also changed the url back to zm(I was using stage)

So I had to change it back to stage and was able to login successfully.

In conclusion; the url is also changed to the default production environment.
